### PR TITLE
[all]: Centralize release and release history

### PR DESCRIPTION
**Pull request by CsatariGergely**
Closes: #3076

### CODE REVIEW

**Comment by collivier** [comment]
please rebase it - side effects due  to wiki.opnfv.org redirected to anuket.io

**Review by collivier — APPROVED** [review]

**Review by xudan16 — APPROVED** [review]

**Review by karinesevilla — APPROVED** [review]

File: doc/index.rst [hunk 1]
```diff
@@ -16,4 +16,37 @@ Anuket Specifications
...
+   * - Baraque
+     - 25th Sep 2020
+   * - Elbrus
+     - 29th Jan 2021
+   * - Kali
+     - 1st Jul 2021
+   * - Lakelse
+     - 4th Jan 2022
+   * - Moselle
+     - 21th Jun 2022
```
Comment (pgoyal01) [inline doc/index.rst:51]:
```suggestion
     - 21st Jun 2022
```

**Comment by rgstori** [comment]
@walterkozlowski @jgu17 can you please review this?

**Comment by CsatariGergely** [comment]
This is merge ready can some of the code owners merge this? Merger of this was a pre-requisite for approving Moselle.
@rgstori @collivier @xudan16 @karinesevilla @jgu17 @walterkozlowski @steelescotr

**Pull request merged**